### PR TITLE
Update Chain ID Shimmer EVM Testnet in networks.json

### PR DESCRIPTION
**Pull request by Phyloiota**
The Testnet was reset and moved to Chain ID 1072. ID 1071 does not exist any longer. Submitted new Chain ID and Multical contract

Fixes # .

Changes proposed in this pull request:
-  new Chain ID
- new Multicall contract
-

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @Phyloiota 

looks like this RPC node is not a full archive node? can check?
I get this error
```
fullArchiveNode Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":44,\"error\":{\"code\":-32000,\"message\":\"root commitment '2d168960809f4582d3b6a9ce04798e2e18b7146f' does not exist\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x751d21047c116413895c259f3f305e38c10b7cf6\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://json-rpc.evm.testnet.shimmer.network/", code=SERVER_ERROR, version=web/5.6.0)
````

File: src/networks.json [hunk 1]
```diff
@@ -2397,4 +2397,4 @@
     "start": 7282345,
```
Comment (ChaituVR) [inline src/networks.json:2397]:
```suggestion
    "start": 10614,
```

File: src/networks.json [hunk 2]
```diff
@@ -1195,14 +1195,14 @@
...
     "name": "Shimmer EVM Testnet",
     "shortName": "ShimmerEVM",
-    "chainId": 1071,
+    "chainId": 1072,
     "network": "testnet",
     "testnet": true,
-    "multicall": "0xb6d9a0849bA7a6FB565567A6013cFEb8d91A1688",
+    "multicall": "0x751d21047C116413895c259f3f305e38C10B7cF6",
     "rpc": [
       "https://json-rpc.evm.testnet.shimmer.network/"
```
Comment (Phyloiota) [inline src/networks.json:1207]:
```suggestion
      "https://archive.evm.testnet.shimmer.network/v1/chains/rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3/evm"
```

**Review by Phyloiota — COMMENTED** [review]
Hey, sorry was not following up here. This is the correct archive node now

**Review by ChaituVR — APPROVED** [review]

**Comment by ChaituVR** [comment]
Hi @Phyloiota This will be updated in the next update. (may take some time)

**Pull request merged**